go.mod

```diff
@@ -12,5 +12,6 @@ require (
 	github.com/kr/pretty v0.3.0 // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/rogpeppe/go-internal v1.6.1 // indirect
+	github.com/sirupsen/logrus v1.9.0 // indirect
```

Comment:
Drive-by: This is a no go for the library, along with outputting to stdio in general.